CREATE-PR.md

```diff
@@ -0,0 +1,12 @@
+# Pushing your article for review
+
+Once you decided to write an article, you can start working directly into the repository by creating a markdown formatted file in `content/<article-title> - <author-name>.md` and start writing!
+
+When you think your article is ready to be reviewed, you can open up a Pull Request on the repo. The same as when creating an issue, you will have a basic template to follow and include some extra information about your article. It’s very important to include a reference to the originating issue to be able to keep a trace of the work being done on the blog.
+
+At that point, one or more content reviewers will be assigned by the blog maintainers to the article. They will take care of reviewing the technical content of the article, making sure everything makes sense and nothing important is missing.
+Once the content is reviewed and considerd ready for publishing, one last review will be done by Austin, who will make sure the grammar, structure and story telling of the article is top notch, taking care of enhancing it's readadbility and appeal.
```

Comment:
The style review is done by Andrea. Maybe we can omit the name and use "copywriter" or something like that